lib/_emerge/main.py

```diff
@@ -1257,6 +1259,10 @@ def emerge_main(args=None):
     os.umask(0o22)
     emerge_config = load_emerge_config(action=myaction, args=myfiles, opts=myopts)
 
+    if myaction == "version":
+        writemsg_stdout(emerge_version(emerge_config) + "\n", noiselevel=-1)
```

Comment:
Could be cool to make noiselevel an enumeration

Reply:
I think you're right, although it's likely to be a job to do in a followup (and quite a big one, even if it is mostly mechanical).